### PR TITLE
update project logs api to v2.0

### PR DESCRIPTION
**Pull request by wy65701436**
use audit log api to get project logs

Signed-off-by: wang yan <wangyan@vmware.com>

### CODE REVIEW

File: api/v2.0/swagger.yaml [hunk 1]
```diff
@@ -615,6 +615,68 @@ paths:
...
+      description: Get recent logs of the projects
+      tags:
+        - project
+      operationId: getLogs
+      parameters:
+        - $ref: '#/parameters/projectName'
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/page'
+        - $ref: '#/parameters/pageSize'
+        - name: username
```
Comment (ywk253100) [inline api/v2.0/swagger.yaml]: Use the new query instead?

File: src/server/v2.0/handler/project.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import (
+	"context"
+	"github.com/go-openapi/runtime/middleware"
+	api_project "github.com/goharbor/harbor/src/api/project"
+	"github.com/goharbor/harbor/src/common/rbac"
+	"github.com/goharbor/harbor/src/pkg/audit"
+	"github.com/goharbor/harbor/src/pkg/q"
+	"github.com/goharbor/harbor/src/server/v2.0/models"
+	"github.com/goharbor/harbor/src/server/v2.0/restapi/operations/project"
+	operation "github.com/goharbor/harbor/src/server/v2.0/restapi/operations/project"
```
Comment (heww) [inline src/server/v2.0/handler/project.go]: duplicated

File: api/v2.0/swagger.yaml (hunk 1 above)
Comment (renmaosheng) [reply api/v2.0/swagger.yaml]: it this like audit log? not just push, delete, also the pull event will be recorded? calling event would be more appropriate.

Comment (wy65701436) [reply api/v2.0/swagger.yaml]: update to Q in query

File: src/server/v2.0/handler/project.go (hunk 2 above)
Comment (wy65701436) [reply src/server/v2.0/handler/project.go]: updated.

File: api/v2.0/swagger.yaml (hunk 1 above)
Comment (ywk253100) [reply api/v2.0/swagger.yaml]: So the query strings username/resource/resource_type/etc can be removed?

**Pull request merged**